docs/core_systems/service_registry.md

```diff
@@ -0,0 +1,202 @@
+:material-file-document: [Abstract System Description (SysD)](../assets/sysd/5_0_0/serviceregistry_sysd.pdf)
+
+## Services
+
+### service-discovery
+
+The purpose of this service is to lookup, register and revoke provided services. The service is offered for both application and core/support systems.
+
+Learn more: <br />
+:material-file-document: [Abstract Service Description (SD)](../assets/sd/5_0_0/service-discovery_sd.pdf) <br />
+:material-api: [generic-http (IDD)](todo) | [generic-https (IDD)](todo) <br />
+:material-api: [generic-mqtt (IDD)](todo) | [generic-mqtts (IDD)](todo) <br />
+:material-tag: since: v5.0.0 
+
+**register**
+
+This service operation adds new service instance to the Local Cloud.
+
+:material-arrow-right-thin: Example: [generic-http](todo) | [generic-https](todo)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+**revoke**
+
+This service operation removes a service instance from the Local Cloud.
+
+:material-arrow-right-thin: Example: [generic-http](todo) | [generic-https](todo)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+**lookup**
+
+This service operation lists the service instances that match the filtering requirements.
+
+:material-arrow-right-thin: Example: [generic-http](todo) | [generic-https](todo)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+-----
+
+### system-discovery
+
+The purpose of this service is to lookup, register and revoke systems that are part of (or want to be part of) the Local Cloud. The service is offered for both application and core/support systems.
+
+Learn more: <br />
+:material-file-document: [Abstract Service Description (SD)](../assets/sd/5_0_0/system-discovery_sd.pdf) <br />
+:material-api: [generic-http (IDD)](todo) | [generic-https (IDD)](todo) <br />
+:material-api: [generic-mqtt (IDD)](todo) | [generic-mqtts (IDD)](todo) <br />
+:material-tag: since: v5.0.0 
+
+**register**
+
+This service operation adds new system to the Local Cloud.
+
+:material-arrow-right-thin: Example: [generic-http](todo) | [generic-https](todo)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+**revoke**
+
+This service operation removes a system from the Local Cloud.
+
+:material-arrow-right-thin: Example: [generic-http](todo) | [generic-https](todo)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+**lookup**
+
+This service operation lists the systems that match the filtering requirements.
+
+:material-arrow-right-thin: Example: [generic-http](todo) | [generic-https](todo)<br />
+:material-arrow-right-thin: Example: [generic-mqtt](todo) | [generic-mqtts](todo)
+
+-----
+
+### device-discovery
+
+The purpose of this service is to lookup, register and revoke devices on which the Local Cloud’s systems are running. The service is offered for both application and core/support systems.
```

Comment:
`devices on which the Local Cloud’s systems are running` 
I would say _can run_ instead of _running_ (because you can perform operations with devices that are not assigned to any system at all)